charts/oidc-webhook-authenticator/values.yaml

```
@@ -24,10 +24,18 @@ webhookConfig:
         -----END RSA PRIVATE KEY-----
 
 # Kubeconfig to the target cluster. In-cluster configuration will be used if not specified.
-# If serviceAccountTokenVolumeProjection.enabled is set to `true` then this value is required.
+# If automountServiceAccountToken is set to `true` then this value is required because in-cluster configuration will not work.
 kubeconfig: 
 
-authKubeconfig: 
+auth:
+  # Kubeconfig to the target authentication and authorization cluster. In-cluster configuration will be used if not specified.
+  # If automountServiceAccountToken is set to `true` then this value is required because in-cluster configuration will not work.
+  kubeconfig:
+  # Specifies if rolebinding to role extension-apiserver-authentication-reader in kube-system namespace and clusterrolebinding to cluster role system:auth-delegator
+  # should be created for the webhook's service account. Usually there is no need for this value to be `false`.
+  installDelegatorBindings: true
+  # A list of HTTP paths to skip during authorization.
+  authorizationAlwaysAllowPaths: /healthz
```

As this is a list, shouldn't it be 
```suggestion
  authorizationAlwaysAllowPaths: 
  - /healthz
```

Then, in the deployment manifest you can use 
```yaml
        - --authorization-always-allow-paths={{ .Values.auth.authorizationAlwaysAllowPaths | join "," }}
```

to join them.

---

You are right. I did the necessary change.